rules-reviewed/eap6/websphere/websphere-catchall.windup.xml

```diff
@@ -8,11 +8,11 @@
             classifications added to them.
         </description>
         <dependencies>
-            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.3.0.Final" />
-            <addon id="org.jboss.windup.rules,windup-rules-java,2.3.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.7.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.7.0.Final" />
         </dependencies>
         <sourceTechnology id="websphere" />
-        <targetTechnology id="eap" versionRange="[6,7)" />
+        <targetTechnology id="eap" versionRange="[6,7]" />
```

Comment:
@d-s also pointed out that this should be [6,8) in order to cover minor versions (for example, 7.1). I think this will need to be changed as well.